src/python/pants/backend/go/util_rules/analyze_package.go

```diff
@@ -120,6 +128,101 @@ func cleanStringSet(valuesMap map[string]bool) []string {
 	return values
 }
 
+func computeEmbedConfigs(directory string, pkg *Package) error {
+	// Obtain a list of files in and under the package's directory. These will be embeddable files.
+	// TODO: Support resource targets elsewhere in the repository.
+
+	fmt.Fprintf(os.Stderr, "computeEmbedConfigs(directory=%s)\n", directory)
```

Comment:
This is a debugging print and will need to come out (along with other instances).